src/Dependencies/Roslyn/project.json

```diff
@@ -1,7 +1,8 @@
 {
   "supports": {},
   "dependencies": {
-    "Microsoft.VisualStudio.LanguageServices": "2.0.0-beta5-60909-09"
+    "Microsoft.VisualStudio.LanguageServices": "2.0.0-beta5-60909-09",
+    "System.ValueTuple": "4.0.1-beta-24425-02"
```

Comment:
I think we should make sure the stable 4.3.0 is being used by VS and use that and not add another thing that needs to change between now and RTM for that.